### DISCUSSION TITLE
Missing "Delete" button for releases

### DISCUSSION THREAD

**Discussion by pmeier** [community/community#23222] [New to GitHub]
<p>I’m trying to delete a release, but I’m not able to. In <a href="https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository#deleting-a-release" rel="nofollow noopener">this tutorial</a> there is a button titled “Delete” next to “Edit release”, but for me this button does not exist</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/6c2f760da560590ebb5b45893d595aae2a19ea9a.png" data-download-href="/uploads/short-url/fr3fC5x6qwzs0sP0tCi1R9ttSpI.png?dl=1" title="Screenshot from 2020-07-10 10-47-41" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/6849766/181075646-d29dbbac-71b6-453c-81b5-fd7bd21dfc99.png" alt="Screenshot from 2020-07-10 10-47-41" data-base62-sha1="fr3fC5x6qwzs0sP0tCi1R9ttSpI" width="690" height="228" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/6/6c2f760da560590ebb5b45893d595aae2a19ea9a_2_690x228.png, https://aws1.discourse-cdn.com/github/original/2X/6/6c2f760da560590ebb5b45893d595aae2a19ea9a.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/6/6c2f760da560590ebb5b45893d595aae2a19ea9a.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/6c2f760da560590ebb5b45893d595aae2a19ea9a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot from 2020-07-10 10-47-41</span><span class="informations">921×305 19.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Am I missing something here?</p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Hi <a class="mention" href="/pmeier">@pmeier</a>! Thank you for being here.  Usually you <a href="https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository#deleting-a-release">must remove all binary files attached to a release before you can delete</a>. Please let us know if you have already tried that step.</p>

**Comment by pmeier** [comment]
<p>Hi <a class="mention" href="/nyahbhinghiprincess">@nyahbhinghiprincess</a></p>
<p>There were no binaries attached. Or does this also applies to the auto-generated “Source Code (zip / tar.gz)” files?</p>

**Comment by TheMostDiligent** [comment]
<p>I am facing the exact same issue. Just wondering why there was no follow-up from <a class="mention" href="/nyahbhinghiprincess">@nyahbhinghiprincess</a> as there seem to be no way to delete auto-generated archives.</p>

**Comment by TheMostDiligent** [comment, marked as answer]
<p>I just accidentally found the answer to this question, so I will post it here:<br>
You need to click on the release name link (v0.1.0 on the screenshot above). Then there will be another page that looks very similar, but does have the Delete button.</p>

**Comment by ddanboyle** [comment]
Actually the GUI for delete has gotten weirder...
You need to click on the release name to get to a new screen, then you have a weird GUI that pops up
![image](https://user-images.githubusercontent.com/1766611/183463566-6ca4cc71-0950-4b34-9d55-5a7ceae185e1.png)
You have to click the word "details" to pull up a dropdown which has a delete button in it.
The arrow to the left goes somewhere else when you click it
The word details isn't highlighted like a link, so you can't tell by looking.
But then when you click Details, a mini menu pops up with a trash can icon.

Git hub really needs a consistent GUI
